ci_group/revolve2/ci_group/morphological_novelty_metric/_morphological_novelty_metric.py

```diff
@@ -0,0 +1,121 @@
+from dataclasses import dataclass, field
+from math import atan2, pi, sqrt
+
+import numpy as np
+from numpy.typing import NDArray
+
+from revolve2.modular_robot import ModularRobot
+
+from ._coordinate_operations import CoordinateOperations
+
+try:
```

Comment:
why this try catch? shouldnt make a difference if you just reraise i think?

Reply:
this is for sphinx docs to not break. Since it cant resolve the so imports.